### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by kwonjaeuk**
Created CODE_OF_CONDUCT.md file to Pluna project.

### CODE REVIEW

**Review by woojeonghyuk — COMMENTED** [review]
good

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Code of Conduct
```
Comment (woojeonghyuk) [inline CODE_OF_CONDUCT.md:1]: good

**Review by Inyong-Hwang — COMMENTED** [review]
Good

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Examples of representation include:
+
+Using an official project email address
+Posting via an official social media account
+Acting as a designated representative at online or offline events
+Behavior outside project spaces may also be addressed if it negatively impacts the project or community.
+
+## Enforcement  
+
+Instances of abusive, harassing, or otherwise unacceptable behavior can be reported by contacting the project team at [conduct@yourproject.com]. All complaints will be reviewed and investigated promptly and fairly.
```
Comment (JunheePark0224) [inline CODE_OF_CONDUCT.md:47]: You should modify e-mail.

**Review by JunheePark0224 — APPROVED** [review]

**Pull request merged**